format/format.go

```diff
@@ -204,9 +204,15 @@ func (w *writer) writeRule(rule *ast.Rule, comments []*ast.Comment) []*ast.Comme
 	if rule.Default {
 		w.write("default ")
 	}
-
 	comments = w.writeHead(rule.Head, comments)
-	if len(rule.Body) == 0 {
+
+	// OPA transforms lone bodies like `foo = {"a": "b"}` into rules of the form
+	// `foo = {"a": "b"} { true }` in the AST. We want to preserve that notation
+	// in the formatted code instead of expanding the bodies into rules, so we
+	// pretend that the rule has no body in this case.
+	isExpandedConst := rule.Head.DocKind() == ast.CompleteDoc && len(rule.Body) == 1 && rule.Body[0].Equal(ast.NewExpr(ast.BooleanTerm(true)))
```

Comment:
Nit: instead of 

```
len(rule.Body) == 1 && rule.Body[0].Equal(ast.NewExpr(ast.BooleanTerm(true)))
```

just

```
rule.Body.Equal(ast.NewBody(ast.NewExpr(ast.NewTerm(true)))
```